### PR TITLE
Added counter when cooking more then 1 meal

### LINKED ISSUE #323: Kitchen fixes
**Description**
There are still some things to do regarding the kitchen

**Subtasks**
- [ ] The smoke and sound effect properly turns off on round over
- [ ] Some kind of indicator as to how many orders are currently being cooked?

### PR DESCRIPTION
**Pull request by tzaarela**
Closes #323 

**Description**  
Added counter when cooking more then 1 meal.

**List of changes**  
- Displaying counter on more then 1 meal

### CODE REVIEW

File: Assets/00_Content/Scripts/Interactables/Kitchens/Kitchen.cs [hunk 1]
```diff
@@ -75,6 +86,9 @@ public class Kitchen : Interactable {
 		private void FinishCooking() {
 			Destroy(tickets.Dequeue());
 
+			if (tickets.Count == 1)
+				cookingCounter.Disable();
```
Comment (MrJackey) [inline Assets/00_Content/Scripts/Interactables/Kitchens/Kitchen.cs:90]: You should probably update the counter here as well, currently it does not decrease when something is finished.

**Review by MrJackey — CHANGES_REQUESTED** [review]
Spotted an error you should take a look at. Also about the position of the counter, would it be better to have it closer to the progressbar and maybe a bit smaller? It feels a bit disconnected from the progress bar as it right now. Perhaps something like this: 
image

File: Assets/00_Content/Scripts/Interactables/Kitchens/Kitchen.cs (hunk 1 above)
Comment (tzaarela) [reply Assets/00_Content/Scripts/Interactables/Kitchens/Kitchen.cs:90]: im not incrementing the counter, im just setting the value from tickets.Count()

Comment (tzaarela) [reply Assets/00_Content/Scripts/Interactables/Kitchens/Kitchen.cs:92]: nvm , im stupid 🐰

**Review by MrJackey — APPROVED** [review]
:shipit:

**Pull request merged**